dashboard/src/assets/constants/overviewConstants.js

```diff
@@ -4,11 +4,9 @@ export const ROWS_PER_PAGE = 5;
 export const DASHBOARD_SAVED = "global.dashboard.saved";
 export const DASHBOARD_SEEN = "global.dashboard.seen";
```

Comment:
Are these constants still used with the change to `global?.dashboard?.saved`?

Reply:
These are being used in `metaDataActions` which is sent as payload for updating the dataset